common/table.js

```diff
@@ -0,0 +1,129 @@
+(function () {
+    'use strict';
+
+    angular.module('chaise.record.table', [])
+
+    .directive('recordTable', ['UriUtils', '$rootScope', '$window', '$filter', function(UriUtils, $rootScope, $window, $filter) {
+        var postLink = function(scope) {
+            var context = $rootScope.context;
+
+            scope.tableModel = {
+                tableName:          null,
+                tableDisplayName:   null,
+                columns:            [],
+                filter:             null,
+                sortedBy:           null,
+                sortOrder:          null,
+                rowset:             null,
+                keycols:            [],
+                count:              0
+            };
```

Comment:
General programming question: I've always wondered, when is it good to list out things and space them out such that all the values in an object are aligned, such as in `scope.tableModel`? Is it just a matter of preference?